client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -195,12 +225,33 @@ const getFormSettings = settings => {
 	] );
 };
 
-const connectComponent = connect( state => {
+const connectComponent = connect( ( state, ownProps ) => {
+	const siteId = getSelectedSiteId( state );
+
+	let podcastingCategoryId = null;
+
+	// Retrieve podcasting category ID from saved category slug (if any)
+	const podcastingCategorySlug = get( ownProps.fields, 'podcasting_archive' );
+	if ( podcastingCategorySlug && podcastingCategorySlug !== '0' ) {
+		const categories = getTerms( state, siteId, 'category' );
+		if ( categories && categories.length ) {
+			// This is not a bound selector
+			// eslint-disable-next-line wpcalypso/redux-no-bound-selectors
```

Comment:
This lint rule fails without this annotation.  It looks like a false positive:  https://github.com/Automattic/eslint-plugin-wpcalypso/pull/38